libs/common/src/lib/resources/edit-resource/profile/profile.component.html

```diff
@@ -177,7 +177,7 @@
   <div class="right-panel-container">
     <ncom-hint
       label="How to modify a resource with the API"
-      learnMore="https://docs.nuclia.dev/docs/api#operation/Modify_Resource_kb__kbid__resource__rid__patch"
+      learnMore="https://docs.nuclia.cloud/docs/api/#tag/Resources/operation/Modify_Resource__by_id__kb__kbid__resource__rid__patch"
```

Comment:
this link with `cloud` doesn't work while it's working when we put back `dev`
```suggestion
      learnMore="https://docs.nuclia.dev/docs/api/#tag/Resources/operation/Modify_Resource__by_id__kb__kbid__resource__rid__patch"
```